lib/core/src/lib/notifications/components/notification-history.component.scss

```diff
@@ -69,3 +69,8 @@
         padding: 0;
     }
 }
+
+.mat-list-base .mat-list-item.adf-notification-history-menu-no-message .mat-line,
```

Comment:
Small comment but that can be a bit shorter: 
```scss
.mat-list-base .mat-list-item {
  &.adf-notification-history-menu-no-message .mat-line, .adf-notification-history-menu-message.mat-line {
    white-space: normal;
  }
}
```

Thanks that you don't need to repeat .mat-list-base .mat-list-item twice.

Reply:
the path for the css is unreasonably long, just create a dedicated class for the mat-line in the Notification component, and style it directly, i.e. 

```css
.adf-notification-message-line { white-space: normal; }
```